edu-services/gradebook-service/impl/src/java/org/sakaiproject/component/gradebook/GradebookServiceHibernateImpl.java

```diff
@@ -323,11 +324,14 @@ public Object doInHibernate(final Session session) throws HibernateException {
 						gradeDef.setGrade(null);
 						gradeDef.setGraderUid(null);
 						gradeDef.setGradeComment(commentText);
+						gradeDef.setExcused(false);
 					} else {
 						gradeDef.setDateRecorded(gradeRecord.getDateRecorded());
 						gradeDef.setGraderUid(gradeRecord.getGraderId());
 						gradeDef.setGradeComment(commentText);
 
+						gradeDef.setExcused(BooleanUtils.toBoolean(gradeRecord.isExcludedFromGrade()));
```

Comment:
This BooleanUtils call could go in the setter instead and then you don't need to worry about calling every time you pass in a boolean value from an "untrusted" source